doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -321,19 +413,19 @@ The list of operators (and the format of the values they use) is as follows:
       be used with attributes which contain dates or numbers. E.g. `temperature!=10..20` matches if
       the temperature is lower than 10 or greater than 20 (it will not match if the tempeature is 10 or the
       temperature is 20).
-+ **Greater than**: `&gt;`. The value is a single element, e.g. `temperature&gt;40`. It
++ **Greater than**: `>`. The value is a single element, e.g. `temperature&gt;40`. It
```

Comment:
Note that the line has two occurrences of the `&gt;` token. The first one has been fixed, but not the second (same in 3 other lines, below).